src/main.cpp

```diff
@@ -39,6 +39,14 @@ struct wf_recorder_output
 
 std::vector<wf_recorder_output> available_outputs;
 
+static timespec get_ct()
```

Comment:
is this function used at all?

Reply:
It's used to output some info but that line could be removed with the entire function.
`std::cout << "start at " << timespec_to_usec(get_ct()) / 1.0e6<< std::endl;`

Reply:
yeah that was a debug print, we don't need it

Reply:
Ok fixed, I just merged everything into a single commit.